bin/webpack-helpers.js

```diff
@@ -127,22 +127,36 @@ const mainEntry = {
 	'panel-style': './node_modules/@wordpress/components/src/panel/style.scss',
 	'custom-select-control-style':
 		'./node_modules/@wordpress/components/src/custom-select-control/style.scss',
+};
 
-	// cart & checkout blocks
+const experimentalMainEntry = {
 	cart: './assets/js/blocks/cart-checkout/cart/index.js',
 	checkout: './assets/js/blocks/cart-checkout/checkout/index.js',
 };
 
-const frontEndEntry = {
+const mainEntry =
+	process.env.WOOCOMMERCE_BLOCKS_PHASE === 'experimental'
+		? Object.assign( {}, stableMainEntry, experimentalMainEntry )
+		: Object.assign( {}, stableMainEntry );
```

Comment:
You can just use object spread here if you want. But this also is okay.

Reply:
forget that was a thing 🤦🏻‍♂️